scrapers/scraper.js

```diff
@@ -342,11 +342,88 @@ function iitdScraper() {
     });
 }
 
+function iitb_ee_Scraper() {
+    var options = {
+        url: 'https://www.ee.iitb.ac.in/web/people/faculty',
+        method: 'GET',
+    //    proxy: 'http://10.8.0.1:8080',
+        headers: {
+            'Connection': 'keep-alive',
+            'Accept': '*/*',
+            'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.114 Safari/537.36'
+        }
+    };
 
+    request(options, function (error, response, html) {
+        if (!error && response.statusCode == 200) {
+            
+          const $ = cheerio.load(html);
+    
+          var fac = $(".facultyview").children('li');
+          fac.each(function (i, el) {
+                var name = $(this).find('a').text().replace(/\s\s+/g,'');
+                name=name.replace(/... more/,'');
```

Comment:
add spaces like `name = name.replace(/... more/,'');` , do this for all variables.

Reply:
@Kriti021999  What about this one??